### ISSUE TITLE
Formatted Text Area doesn't read label and hint

### ISSUE THREAD

**Issue by ebruchez** [orbeon/orbeon-forms#5354]
See #4832. The label and hint are there, but are not linked to anything.

Possibly adding `role="group"` and `labelled-by`/`described-by` on the XBL element will work?
